docs/readmes/README.role-assignment.md

```diff
@@ -2,6 +2,22 @@
 
 AKS and aks-engine clusters require an identity to communicate with Azure. This identity can be either a **managed identity** (in the form of system-assigned identity or user-assigned identity) or a **service principal**. This section explains various role assignments that need to be performed before using AAD Pod Identity. Without the proper role assignments, your Azure cluster will not have the correct permission to assign and un-assign identities from the underlying virtual machines (VM) or virtual machine scale sets (VMSS).
 
+## TL;DR
+
+Run the following commands to help you set up the apprioapate role assignments for your cluster identity before deploying aad-pod-identity (assuming you are running an AKS cluster):
+
+```bash
+export SUBSCRIPTION_ID="<SubscriptionID>"
+export RESOURCE_GROUP="<AKSResourceGroup>"
```

Comment:
Let's add a note here for what the AKSResourceGroup here is. In the demo we mention using `MC_` as AKSResourceGroup and that conflicts with the same variable here.